Meadowlark-js/docker-meadowlark/docker-compose.yml

```diff
@@ -5,12 +5,11 @@ services:
     container_name: opensearch-node1
     environment:
       cluster.name: opensearch-cluster
-      bootstrap.memory_lock: true # along with the memlock settings below, disables swapping
+      bootstrap.memory_lock: "true" # along with the memlock settings below, disables swapping
       OPENSEARCH_JAVA_OPTS: -Xms512m -Xmx512m # minimum and maximum Java heap size, recommend setting both to 50% of system RAM
       discovery.type: single-node
-      DISABLE_INSTALL_DEMO_CONFIG: true # disables execution of install_demo_configuration.sh bundled with security plugin, which installs demo certificates and security configurations to OpenSearch
-      DISABLE_SECURITY_PLUGIN: true # disables security plugin entirely in OpenSearch by setting plugins.security.disabled: true in opensearch.yml
-    mem_limit: 2g
```

Comment:
I changed these because they were erroring out on linux.

`brad@ubuntu:~/work/meadowlark/Meadowlark/Meadowlark-js/docker-meadowlark$ sudo docker-compose up -d
ERROR: The Compose file './docker-compose.yml' is invalid because:
Unsupported config option for services.opensearch-node1: 'mem_limit'
services.opensearch-dashboards.environment.DISABLE_SECURITY_DASHBOARDS_PLUGIN contains true, which is an invalid type, it should be a string, number, or a null
services.opensearch-node1.environment.bootstrap.memory_lock contains true, which is an invalid type, it should be a string, number, or a null`

Reply:
Ok, I figured it out. On linux right now the new docker compose plugin is a manual install. Once I added that, `docker compose` was available as is fine with original docker-compose.yml file.